CHANGES.md

```diff
@@ -11,8 +11,10 @@
 
 * filter, frequencies, refine: Speed up reading of the metadata file. [#1252][] (@victorlin)
 * traits: Previously, columns with only numeric values were treated as numerical data. These are now treated as categorical data for discrete trait analysis. [#1252][] (@victorlin)
+* Enforce UTF-8 encoding when reading and writing files. Improve error messages when a non-UTF-8 file is used. [#1381][]
```

Comment:
Also worth noting that many more input files and output files now support compression?

Reply:
Good point, will add this after updating calls to `pandas.read_csv()`.

Reply:
[Added](https://github.com/nextstrain/augur/compare/c2ff05e9acc8426db466cd7010572b4364288e07..b695954309a16e53d315e017f6601f2d5f886148):

> Compression should now be supported for all input and output files. Please [open an issue](https://github.com/nextstrain/augur/issues) if you find one that doesn't!